style.css

```diff
@@ -0,0 +1,717 @@
+    margin-left: 10px;
+}
+.btn—prev,
+.btn—next {
+    background-color: transparent;
+    text-decoration: none;
+    color: #F5F8FB; 
+    border-width: 2px;
+    border-style: solid;
+    border-color: rgba(208, 215, 251, 0.17);
+    height: 48px;
+    width: 48px;
+    align-items: center;
+    justify-content: center;
+}
+.btn—prev:hover,
+.btn—next:hover {
+    background-color: #0265DC;
+}
+.btn—prev::after {
+    content: "<";
+}
+.btn—next::after {
+    content: ">";
+}
+.catalog__list {
+    gap: 10px;
+    display: flex;
+}
+.catalog__item {
+    width: 300px;
+    background-color: #1D1C28;
+    padding: 16px 22px 16px 16px;
+}
+.catalog__image img {
+    width: 100%;
+    height: auto;
+}
+.catalog__image{
+    position: relative;
+}
+.catalog__label {
+    position: absolute;
+    top: 8px;
+    right: 3px;
+    background-color: #F5F8FB;
+    color: #0265DC;
+    font-size: 14px;
+    font-weight: 600;
+    padding: 4px 8px;
+    text-transform: uppercase;
+}    
+.catalog__label--red {
+    background-color: #EA3829;
+    color: #F5F8FB;
+}
+.catalog__price {
+    display: block;
+    font-family: "Bakbak One", Times New Roman, Georgia, serif;
+    font-size: 36px;
+    line-height: 48px;
+}
+.catalog__heading {
+    display: block;
+    color: #EFF5FA;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+}
+.section-title {
+    text-align: center;
+    font-weight: 400;
+    color: #F8F7FF;
+    margin-bottom: 25px;
+}
+.blog {
+    margin: 0 auto;
+    max-width: 1758px;
+    padding: 0 15px;
+    margin-bottom: 100px;
+}
+.blog__list {
+    display: flex;
+    gap: 10px;
+    flex-wrap: wrap;
+    margin-bottom: 40px;
+}
+.blog__item {
+    background-color: #1D1C28;
+    padding: 16px;
+    flex-basis: calc(50% - 10px);
+    flex-shrink: 1;
+    color: rgba(239, 245, 250, 0.82);
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+}
+.blog__img {
+    position: relative;
+    margin-bottom: 15px;
+}
+.blog__img img {
+    max-width: 100%;
+    height: auto;
+}
+.blog__label {
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    color: #0265DC;
+    font-size: 14px;
+    font-weight: 600;
+    line-height: 20px;
+    background-color: #F5F8FB;
+    height: 36px;
+    padding: 8px 10px;
+}
+.blog__info {
+    display: flex;
+    margin-bottom: 10px;
+    font-size: 14px;
+    line-height: 20px;
+    font-weight: 600;
+}
+.blog__autor {
+    margin-right: 20px;
+}
+.blog__autor::after {
+    content: "";
+    position: absolute;
+    top: 2px;
+    right: -14px;
+    z-index: 2;
+    width: 2px;
+    height: 16px;
+    background-color: rgba(208, 215, 251, 0.17);
+}
+.blog__autor,
+.blog__info time {
+    position: relative;
+    padding-left: 26px;
+}
+.blog__autor::before,
+.blog__info time::before {
+    content: "";
+    position: absolute;
+    top: 2px;
+    left: 4px;
+    z-index: 2;
+    width: 16px;
+    height: 16px;
+    background-image: url(img/user.svg);
```

Comment:
If code has different images for different elements, I suggest split classes. 
Because otherwise we will have an image redefinition for one of them, which should not have been there. We don't need to redefine what shouldn't be